tests/bofire/benchmarks/test_aspen_benchmark.py

```diff
@@ -0,0 +1,174 @@
+            ]
+        ),
+        output_features=OutputFeatures(
+            features=[
+                ContinuousOutput(
+                    key="X",
+                    type="ContinuousOutput",
+                    objective=MaximizeObjective(
+                        type="MaximizeObjective", w=1.0, lower_bound=0, upper_bound=1
+                    ),
+                ),
+                ContinuousOutput(
+                    key="Y",
+                    type="ContinuousOutput",
+                    objective=MinimizeObjective(
+                        type="MinimizeObjective", w=1.0, lower_bound=0, upper_bound=1
+                    ),
+                ),
+                ContinuousOutput(
+                    key="Z",
+                    type="ContinuousOutput",
+                    objective=MinimizeObjective(
+                        type="MinimizeObjective", w=1.0, lower_bound=0, upper_bound=1
+                    ),
+                ),
+            ]
+        ),
+        constraints=Constraints(
+            constraints=[
+                LinearInequalityConstraint(
+                    type="LinearInequalityConstraint",
+                    features=["E", "D"],
+                    coefficients=[1.0, 1.0],
+                    rhs=10.0,
+                ),
+                LinearInequalityConstraint(
+                    type="LinearInequalityConstraint",
+                    features=["C", "A"],
+                    coefficients=[-4.0, -5.0],
+                    rhs=-2.0,
+                ),
+                LinearInequalityConstraint(
+                    type="LinearInequalityConstraint",
+                    features=["A", "D", "C"],
+                    coefficients=[-0.6, 7.0, 5.0],
+                    rhs=-10.0,
+                ),
+                LinearInequalityConstraint(
+                    type="LinearInequalityConstraint",
+                    features=["B", "A"],
+                    coefficients=[-1.0, -1.0],
+                    rhs=-700.0,
+                ),
+            ],
+            experiments=None,
+            candidates=None,
+        ),
+    ),
+    "paths": {
+        "A": "\\Data\\Flowsheeting Options\\Calculator\\CA-01\\Input\\FVN_INIT_VAL\\A",
+        "B": "\\Data\\Flowsheeting Options\\Calculator\\CA-01\\Input\\FVN_INIT_VAL\\B",
+        "C": "\\Data\\Streams\\Stream1\\Input\\TEMP\\C",
+        "D": "\\Data\\Blocks\\DUM-01\\Input\\D",
+        "E": "\\Data\\Blocks\\DUM-02\\Input\\E",
+        "F": "\\Data\\Blocks\\DUM-03\\Input\\F",
+        "G": "\\Data\\Blocks\\DUM-4\\Input\\G",
+        "H": "\\Data\\Flowsheeting Options\\Calculator\\CA-02\\Input\\FVN_INIT_VAL\\H",
+        "X": "\\Data\\Blocks\\DUM-03\\Input\\X",
+        "Y": "\\Data\\Blocks\\DUM-4\\Input\\Y",
+        "Z": "\\Data\\Flowsheeting Options\\Calculator\\CA-02\\Input\\FVN_INIT_VAL\\Z",
+    },
+}
+
+
+@pytest.mark.parametrize(
+    "cls_benchmark, kwargs",
+    [
+        (Aspen_benchmark, input_args),
+    ],
+)
```

Comment:
no need for the parameterization. Just move the stuff from above directly into the test function and call it without arguments.